packages/components/src/Banner/Banner.tsx

```diff
@@ -74,12 +85,37 @@ export function Banner({
     [styles.dismissibleSpacing]: dismissible,
   });
 
+  function getPrev() {
+    current > 0 && setCurrent((current - 1) % messages.length);
+  }
+  function getNext() {
+    setCurrent((current + 1) % messages.length);
+  }
+
   return (
     <>
       {showFlash && (
         <div className={flashClassNames} ref={bannerRef} role="status">
           <div className={contentClassNames}>
-            <Text>{children}</Text>
+            {!multiple && <Text>{children}</Text>}
+            {multiple && (
+              <>
+                <div className={styles.message}>
+                  <Text>{messages[current]}</Text>
+                </div>
+                <div className={styles.pagination}>
+                  <div className={styles.prev} onClick={getPrev}>
+                    <Icon name="arrowLeft" />
+                  </div>
+                  <Text>
+                    {current + 1} of {messages.length}
+                  </Text>
```

Comment:
It seems the text in the banner isn't quite positioned in the center of the banner. Could you do something like: `align-items: center;`?

Reply:
![fix middle](https://user-images.githubusercontent.com/88052089/135181998-a8537bad-20d5-441f-b46e-f2d7984fcbbc.gif)

Reply:
![fix middle](https://user-images.githubusercontent.com/88052089/135182185-434141f3-a5ec-440b-a3ca-d8bfc6b34406.gif)